Buildings/Resources/Documentation/userGuide/source/development.rst

```diff
@@ -170,18 +539,38 @@ The following rules need to be followed, in addition to the guidelines described
    Do not use ``Dialog(tab=STRING))``, unless the parameter is declared with a default value
    and this parameter and its value is of no interest to typical users.
 
-#. Each block must have a ``defaultComponentName`` annotation.
+#. In the code text the instances must be ordered as follows:
+     - If an instance has a type associated with it, then list them in this order: `boolean`, `integer`, `real`.
```

Comment:
I don't understand this. Every instance has a type. Also, use the Modelica types, e.g., `Real` and not `real`.

Reply:
An instance of a subsequence block does not have a type.